### PR TITLE
Update the version for CLI to 1.1.1 from 5.1.0. 

### PR DESCRIPTION
**Pull request by Jagdish-Kewat-EDB**
The CLI version 5.1.0 is released and supported with PGD 5.1.x and not 4.x

For PGD 4.3.1, CLI version 1.1.1 was released.

### CODE REVIEW

File: product_docs/docs/pgd/4/rel_notes/index.mdx [hunk 1]
```diff
@@ -47,3 +47,4 @@ The EDB Postgres Distributed documentation describes the latest version of EDB P
 
 
 [^*] We released a patch to HARP 2.2.1 to address a security vulnerability. If using 2.2.1 with an earlier version of EDB Postgres Distributed, we recommend that you upgrade to 2.2.2.
+q
```
Comment (irionr) [inline product_docs/docs/pgd/4/rel_notes/index.mdx:50]: typo ?

Comment (djw-m) [inline product_docs/docs/pgd/4/rel_notes/index.mdx:50]:
```suggestion
```

Seems to be a loose "q".

File: product_docs/docs/pgd/4/rel_notes/index.mdx [hunk 2]
```diff
@@ -47,3 +47,4 @@ The EDB Postgres Distributed documentation describes the latest version of EDB P
 
 
 [^*] We released a patch to HARP 2.2.1 to address a security vulnerability. If using 2.2.1 with an earlier version of EDB Postgres Distributed, we recommend that you upgrade to 2.2.2.
```
Comment (irionr) [inline product_docs/docs/pgd/4/rel_notes/index.mdx:49]: maybe adjust this too? and recommend 2.2.3?

**Review by irionr — DISMISSED** [review]
LGTM , appart for the typo

**Review by djw-m — CHANGES_REQUESTED** [review]

File: product_docs/docs/pgd/4/rel_notes/index.mdx (hunk 1 above)
Comment (Jagdish-Kewat-EDB) [reply product_docs/docs/pgd/4/rel_notes/index.mdx:50]: Ahh. ok. thanks

File: product_docs/docs/pgd/4/rel_notes/index.mdx (hunk 2 above)
Comment (Jagdish-Kewat-EDB) [reply product_docs/docs/pgd/4/rel_notes/index.mdx:49]: Yes, this would have a separate line for 2.3.0. Creating a separate rel notes for that.

**Review by djw-m — APPROVED** [review]
LGTN

**Review by krupalr-edb — APPROVED** [review]
LGTM

**Pull request merged**